crates/router/src/routes/routing.rs

```diff
@@ -985,33 +985,35 @@ pub async fn toggle_success_based_routing(
     state: web::Data<AppState>,
     req: HttpRequest,
     query: web::Query<api_models::routing::ToggleSuccessBasedRoutingQuery>,
-    path: web::Path<routing_types::ToggleSuccessBasedRoutingPath>,
+    path: web::Path<(
+        common_utils::id_type::MerchantId,
+        common_utils::id_type::ProfileId,
+    )>,
```

Comment:
Now that you are capturing both of these in path itself, can we remove `ToggleSuccessBasedRoutingPath`? if so, please remove it from openapi.rs too